shell/key-bindings.fish

```diff
@@ -69,8 +69,13 @@ function fzf_key_bindings
       # history's -z flag was added in fish 2.4.0, so don't use it for versions
       # before 2.4.0.
       if [ "$FISH_MAJOR" -gt 2 -o \( "$FISH_MAJOR" -eq 2 -a "$FISH_MINOR" -ge 4 \) ];
-        history -z | eval (__fzfcmd) --read0 --print0 -q '(commandline)' | read -lz result
-        and commandline -- $result
+        if type -q perl
+          history -z --reverse | perl -0 -pe 's/^/$.\t/g; s/\n/\n\t/gm' | eval (__fzfcmd) --tac --read0 --print0 -q '(commandline)' | perl -pe 's/^\d*\t//' | read -lz result
```

Comment:
```suggestion
          history -z --reverse | perl -0 -pe 's/^/$.\t/g; s/\n/\n\t/gm' | eval (__fzfcmd) -n2..,.. --tac --read0 --print0 -q '(commandline)' | perl -pe 's/^\d*\t//' | read -lz result
```

Reply:
Ah, okay. But in this way, we can't allow users to override `--nth` in their `$FZF_CTRL_R_OPTS`. Please see 05b4e09.